cmd/katib-controller/v1beta1/main.go

```diff
@@ -32,38 +33,32 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
 
+	configapi "github.com/kubeflow/katib/pkg/apis/config/v1beta1"
 	apis "github.com/kubeflow/katib/pkg/apis/controller"
-	controller "github.com/kubeflow/katib/pkg/controller.v1beta1"
+	"github.com/kubeflow/katib/pkg/controller.v1beta1"
 	"github.com/kubeflow/katib/pkg/controller.v1beta1/consts"
-	trialutil "github.com/kubeflow/katib/pkg/controller.v1beta1/trial/util"
+	"github.com/kubeflow/katib/pkg/util/v1beta1/katibconfig"
 	webhook "github.com/kubeflow/katib/pkg/webhook/v1beta1"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 )
 
+var scheme = runtime.NewScheme()
+
+func init() {
+	utilruntime.Must(apis.AddToScheme(scheme))
+	utilruntime.Must(configapi.AddToScheme(scheme))
+	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
```

Comment:
That's right. Thanks!

Reply:
Through more investigation, we need to add the default K8s API to the scheme.
I faced the below error: `no kind is registered for the type v1.ConfigMap in scheme "pkg/runtime/scheme.go:100"`

Reply:
@tenzen-y I guess, that is because we don't use [scheme from the manager](https://github.com/kubeflow/katib/blob/master/cmd/katib-controller/v1beta1/main.go#L120C33-L120C42) that we are creating, right ?
Is `utilruntime` a new recommended way to setup manager via controller-runtime lib ?

Reply:
> I guess, that is because we don't use scheme from the manager that we are creating, right ?

Yes.

> Is utilruntime a new recommended way to setup manager via controller-runtime lib ?

Yea, this is recommended way: https://book.kubebuilder.io/cronjob-tutorial/main-revisited.html